### ISSUE TITLE
It cannot input chinese character

### ISSUE THREAD

**Issue by acekingke** [danielearwicker/carota#32]
The live demo cannot input chinese character

**Comment by danielearwicker** [comment]
This is true. I only speak/write English so I don't have the necessary knowledge to know if other languages are working correctly, so I hope someone with that knowledge can help.

**Comment by acekingke** [comment]
no about the language, just for Input Method, When the input method is open keycode is 299 and can make it with the charcode

**Comment by danielearwicker** [comment]
If you can make a pull request for this and it doesn't affect functionality in other languages then I will accept it gladly!
